src/ECER.Resources.Documents/PortalInvitations/IPortalInvitationRepository.cs

```diff
@@ -14,10 +17,21 @@ public record PortalInvitation(string? Id, string Name, string ReferenceFirstNam
   public string? WorkexperienceReferenceId { get; set; }
   public string? CharacterReferenceId { get; set; }
   public InviteType? InviteType { get; set; }
+  public PortalInvitationStatusCode? StatusCode { get; set; }
```

Comment:
Returning the status code here is important and valuable - you use it in the manager for validation. Good.